### ISSUE TITLE
Avatar customization

### ISSUE THREAD

**Issue by laughinghan** [ekmartin/slack-irc#86]
Branching out from #42, I'd love to be able to choose to use, say, robohash.org/ instead of avatar.adorable.io because I'm totally creeped out by the weird squished faces that avatar.adorable.io generates, whereas I welcome our robohash overlords. (Someone who really hates happiness might prefer [GitHub's identicons](https://github.com/blog/1586-identicons), etc.)

While we're at it, we should let you disable them entirely so the Slack bot user's default icon shows up.

**Comment by laughinghan** [comment]
So something like:
```js
"avatar_url": "https://robohash.org/$nick.png"
```
and
```js
"avatar_url": false
```
to disable generated avatars.

(What if by some stroke of evil genius, an identicon service provider (ISP?) requires the string `'$nick'` somewhere in their URL? Well their evil plan is foiled, because we can just percent-encode the dollar sign.)

**Comment by laughinghan** [comment]
Ooh also, you know what would be cool, you can set metadata for your nick on NickServ (`/msg nickserv set property avatar http://example.com/avatar.png`), we could query for that when someone joins and use it if present

Maybe that would be a totally useless feature

**Comment by ekmartin** [comment]
I like the idea of something like `"avatarUrl": "https://robohash.org/$username.png"`. Would you want to try and send a pull request? I think the NickServ feature might be a bit too specific .

Robohash was nice though, cool find.

**Comment by RoboMWM** [comment]
> I'm totally creeped out by the weird squished faces that avatar.adorable.io generates

Same here, very odd faces indeed, would rather disable an avatar altogether if possible instead of having this weirdness.

**Comment by ekmartin** [comment]
I don't have time to implement a configuration option just right now, but if anyone's interested in giving it a shot the URL is specified here: https://github.com/ekmartin/slack-irc/blob/master/lib%2Fbot.js#L237

**Comment by ekmartin** [comment]
This has been fixed and released in 3.8.0: https://github.com/ekmartin/slack-irc/blob/master/CHANGELOG.md#380---2016-04-30
